example/example/settings/base.py

```diff
@@ -173,6 +173,8 @@
 GRAPHENE = {"SCHEMA": "grapple.schema.schema"}
 GRAPPLE_APPS = {"images": "", "home": "", "documents": ""}
 GRAPPLE_ADD_SEARCH_HIT = True
+GRAPPLE_PAGE_SIZE = 10
+GRAPPLE_PAGE_SIZE_MAX = 50
```

Comment:
suggestion: worth lowering these both, say 5 and 6 (instead of 10 and 50) so the max page size can be tested

Reply:
Yeah, I'll add some tests later today.